iOS7

jdavis713 opened the pull request:
@bhakes 

I apologize that you're just getting this. I completed (again) my project this morning, working on stretch goals.

erica committed:
Revising README and removing skeleton

jdavis713 committed:
Initial commit

painting model and model controller were created and configured

jdavis713 committed:
controller and UI

I finished settting the controller up and began setting up the UI

jdavis713 committed:
Storyboard reconfigure

reconfigured the storyboard and model controller

jdavis713 committed:
Views and View Controller

Created a PaintingTableViewCell and PaintingListViewController

jdavis713 committed:
Delegates

Configured delegates

jdavis713 committed:
Outlet fix

I deleted an extra outler that prevented my app from building successfully

jdavis713 committed:
initial commit

deleted extraneous swift file. final project built with no bugs.

jdavis713 committed:
Stretch Goal

Added a border and rounded corners of UIImageViews. working on modal segue now.

bhakes commented:
Jordan,

The project builds correctly and implements the MVP.

Also, I LOVE this commit cadence. Keep it up!

Regarding the commit messages, commits should always complete the following sentence:

```
If applied, this commit will <your subject line here>
```

So, for instance, for the last commit, a great commit message would be:
```
Implement Stretch goals
```

Writing good commits is an art! For more info, check this post out: https://chris.beams.io/posts/git-commit/

jdavis713 committed:
Second Stretch Implementation

Implemented a second stretch goal to present an info screen modally and dismiss it when a button is pressed.
